Sources/PrinceOfVersions/PoVDataTypes/PoVRequestOptions.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  PoVRequestOptions.swift
+//  PrinceOfVersions
+//
+//  Created by Ivana Mršić on 20/04/2020.
+//
+
+import Foundation
+
+@objcMembers
+public class PoVRequestOptions: NSObject {
+
+    // MARK: - Public properties
+
+    /// Boolean that indicates whether PoV should use security keys from all certificates found in the main bundle. Default value is `false`.
+    public var shouldPinCertificates: Bool = false
+
+    /// Optional HTTP header fields.
+    public var httpHeaderFields: NSDictionary?
```

Comment:
To disable to adding of keys that are not strings you can do:
```suggestion
    public private(set) var httpHeaderFields: NSDictionary?
    public func set(value: Any, forHttpHeaderField httpHeaderField: String)
```